### PR TITLE
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa…

### PR DESCRIPTION
**Pull request by tibetiroka**
…aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa

### CODE REVIEW

**Comment by warp-core** [comment]
I think your 'a' key is stuck.

**Comment by tibetiroka** [comment]
Oh, that explains it.

File: CreatingGovernments.md [hunk 1]
```diff
@@ -325,7 +325,7 @@ government "Republic"
...
+Beginning in **v. 0.10.0**, governments are allowed to have multiple raid fleets defined, whereas before that point, only a single raid fleet could be defined for each government. Each fleet can also be given a minimum and maximum attraction value for which it will spawn. If a minimum attraction is not provided then the default minimum value of 2 is used. If a minimum attraction is provided but no maximum, then the default maximum value is unbounded (a value of 0 for the maximum value means there is no maximum). This can allow for systems having different raid fleets depending on whom the player is hostile toward, as well as different fleets depending on how attractive the player is to raids.
```
[the remark this answers is not in the document]
Comment (tibetiroka) [reply CreatingGovernments.md:328]: Looking at it, both seems to be correct?

File: CreatingShips.md [hunk 2]
```diff
@@ -224,7 +224,7 @@ The `attributes` key should be followed by a list of ship attributes, ideally li
 
 * `"cargo space"`: the mass of cargo that can be carried.
 
-* `"outfit space"`: the amount of outfits (weapons, generators, engines, etc.) that can be installed.
+* `"outfit space"`: the number of outfits (weapons, generators, engines, etc.) that can be installed.
```
[the remark this answers is not in the document]
Comment (tibetiroka) [reply CreatingShips.md:227]: ~~but outfits is countable~~

Amount really doesn't sound right tbh.

File: CreatingSubstitutions.md [hunk 3]
```diff
@@ -15,7 +15,7 @@ substitutions
 
 # Substitution characteristics
 
-A `substitutions` node can be defined in two places. The first place is as a child of a `missions` node, as described in [creating missions](CreatingMissions#text-replacements). Substitutions defined in this way apply only to the mission they are defined within. The second way to define substitutions is with the `substitutions` node being a root node. Substitutions defined in this was are applied globally to all missions and ship hails.
+A `substitutions` node can be defined in two places. The first place is as a child of a `missions` node, as described in [creating missions](CreatingMissions#text-replacements). Substitutions defined in this way apply only to the mission they are defined within. The second way to define substitutions is with the `substitutions` node being a root node. Substitutions defined in this node are applied globally to all missions and ship hails.
```
[the remark this answers is not in the document]
Comment (tibetiroka) [reply CreatingSubstitutions.md:18]:
```suggestion
A `substitutions` node can be defined in two places. The first place is as a child of a `missions` node, as described in [creating missions](CreatingMissions#text-replacements). Substitutions defined in this way apply only to the mission they are defined within. The second way to define substitutions is with the `substitutions` node being a root node. Substitutions defined in this way are applied globally to all missions and ship hails.
```

File: CreatingGovernments.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (warp-core) [reply CreatingGovernments.md:328]: Maybe

**Pull request merged**